crates/sui-sdk/src/apis.rs

```diff
@@ -364,6 +695,25 @@ impl CoinReadApi {
         )
     }
 
+    /// Return a list of coins for the provided address, or an error upon failure.
```

Comment:
Needs more detail -- this function selects coins to meet or exceed the `amount` balance.